dashboard/test/controllers/api/v1/pd/workshop_survey_report_controller_test.rb

```diff
@@ -251,16 +250,52 @@ class WorkshopSurveyReportControllerTest < ::ActionController::TestCase
     end
 
     test 'experiment_survey_report: return empty result for workshop without responds' do
```

Comment:
This test was using a workshop with no facilitators, which is not a very common case and not explicitly what the test is checking (it cares about no survey responses, not no facilitators).  I've changed the workshop to have facilitators, which also changes the expected result, but I think it still captures the spirit of the test which is an "empty result."  Let me know if you disagree.